stdlib/pathname.rb

```diff
@@ -1,7 +1,14 @@
+require 'corelib/comparable'
+
 class Pathname
+  include Comparable
+
   def initialize(path)
-    raise ArgumentError if path == "\0"
```

Comment:
could you please explain the purpose of this line ? I don't remember exactly but this cause me troubles :neutral_face: